Template/Components/Frame.html

```diff
@@ -1,5 +1,5 @@
         <header id="banner">
-            <img id="banner‐image" src="[*Site Root*]Images/2007-07-24%20header%20image.jpg" alt="Louise Avenue Congregational Church: To know Christ, and to make Him known">
+            <img id="banner‐image" siteSource="Images/2007-07-24%20header%20image.jpg" alt="Louise Avenue Congregational Church: To know Christ, and to make Him known">
```

Comment:
`siteSource` is a pseudo‐attribute that represents a path relative to the site root. It unfolds into a real `src` attribute.